### PR TITLE
Remove unnecessary reflection

### PR DESCRIPTION
**Pull request by cerus**
- Removed unnecessary reflection
- Bumped version to 1.0.1-SNAPSHOT

### CODE REVIEW

File: src/main/java/dev/lukaesebrot/pipe/command/Command.java [hunk 1]
```diff
@@ -141,6 +142,7 @@ public void emit(CommandSender sender, String[] arguments) {
      * @param sender The sender of the tab completion event
      * @param alias  The alias the sender used
      * @param args   The arguments the sender specified
+     *
```
Comment (lus) [inline src/main/java/dev/lukaesebrot/pipe/command/Command.java]: Remove this

File: src/main/java/dev/lukaesebrot/pipe/command/Command.java [hunk 2]
```diff
@@ -101,6 +101,7 @@ public String getName() {
      * Checks whether or not the current command corresponds to the first string of the given array
      *
      * @param strings The array to check
+     *
```
Comment (lus) [inline src/main/java/dev/lukaesebrot/pipe/command/Command.java]: Remove this

File: src/main/java/dev/lukaesebrot/pipe/command/Command.java [hunk 3]
```diff
@@ -68,6 +67,7 @@ public Command(String name,
      * Checks the permissions of the given player with the local {@link PermissionPolicy}
      *
      * @param sender The command sender to check the permissions on
+     *
```
Comment (lus) [inline src/main/java/dev/lukaesebrot/pipe/command/Command.java]: Remove this

**Review by lus — CHANGES_REQUESTED** [review]

**Review by lus — CHANGES_REQUESTED** [review]
No version bumping required, because nothing functional did change.

**Comment by cerus** [comment]
> No version bumping required, because nothing functional did change.

Thats true, but that last digit of the version corresponds to patches, and this is a patch.

**Comment by lus** [comment]
> > No version bumping required, because nothing functional did change.
> 
> Thats true, but that last digit of the version corresponds to patches, and this is a patch.

Thats true, but the whole thing is not released yet, so I don't see any sense in bumping the version.

**Comment by cerus** [comment]
If you don't want to follow [Semantic Versioning](https://semver.org/) I will gladly remove the bump.

**Review by lus — APPROVED** [review]
Seems fine to me^^

Thank you!

**Pull request merged**